jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Stream.java

```diff
@@ -226,6 +227,18 @@ public boolean isLocallyClosed()
         return closeState.get() == CloseState.LOCALLY_CLOSED;
     }
 
+    @Override
+    public void commit()
+    {
+        committed = true;
```

Comment:
what is the memory barrier on this?

Reply:
The flag is set and read from the flusher thread, so the memory barrier is guaranteed by `IteratingCallback`.